packages/tupaia-web/src/constants/constants.ts

```diff
@@ -37,3 +37,15 @@ export const FORM_FIELD_VALIDATION = {
 };
 
 export const PASSWORD_RESET_TOKEN_PARAM = 'passwordResetToken';
+
+export enum TABS {
+  MAP = 'map',
+  DASHBOARD = 'dashboard',
+}
+
+export const MOBILE_THRESHOLD = '900px';
+
+export const URL_SEARCH_PARAMS = {
```

Comment:
I think we could now make a new constant file called url.ts and move all the url related constants into it. What do you think?